webapp/tests/elster_client/mock_erica.py

```diff
@@ -291,6 +295,21 @@ def get_address_data(input_body, show_response: bool):
             else:
                 return get_json_response('get_address_no_resp')
 
+    @staticmethod
+    def is_valid_tax_number(state_abbreviation, tax_number):
+        if err_response := MockErica.errors_from_error_flags(True):
+            return err_response
+
+        _VALID_TAX_NUMBERS = ['19811310010']
+        _VALID_STATE_ABBREVIATIONS = ['BW', 'BY', 'BE', 'BB', 'HB', 'HH', 'HE', 'MV', 'ND', 'NW', 'RP', 'SL', 'SN', 'ST', 'SH', 'TH']
+
+        if MockErica.tax_number_is_invalid \
+                or tax_number not in _VALID_TAX_NUMBERS \
+                or state_abbreviation not in _VALID_STATE_ABBREVIATIONS:
+            return get_json_response('tax_number_is_invalid')
```

Comment:
Isn't `MockErica.tax_number_is_invalid` and `tax_number not in _VALID_TAX_NUMBERS` semantically the same thing? That is, the tax number is not valid - no matter if because of the combination of state and number or just the number?

Reply:
I just think: What is the flag for if we can just pass an invalid tax number? Or vice versa: Why do we need to check the tax number if we can also just set the `tax_number_is_invalid` flag?

Reply:
Sure, I get it. 👍